src/components/player.js

```diff
@@ -32,11 +35,18 @@ class Player extends videojs.getComponent('player') {
     const { kind, language } = trackSelector;
     const audioTrack =
       audioTracks.find((audioTrack) => {
-        return (audioTrack.enabled =
-          audioTrack.language === language && audioTrack.kind === kind);
+        if (audioTrack.language === language && audioTrack.kind === kind) {
+          return (audioTrack.enabled = true);
+        }
+
+        return false;
       }) ||
       audioTracks.find((audioTrack) => {
-        return (audioTrack.enabled = audioTrack.language === language);
+        if (audioTrack.language === language) {
+          return (audioTrack.enabled = true);
+        }
```

Comment:
Is it possible to do:

```javascript
const audioTrack = audioTracks.find((audioTrack) => { audioTrack.language === language && audioTrack.kind === kind}) || audioTracks.find((audioTrack) => { audioTrack.language === language });

audioTrack.enabled = audioTrack !== undefined
```

Reply:
@waliid I really like your suggestion thanks! However, it requires a little adaptation as we're not on Macintosh swift.